### PR TITLE
feat: huly template

### PR DESCRIPTION
**Pull request by bhaleraosaurabh**
1. Added template for huly
2. Updated docs to add huly in templates section

https://github.com/user-attachments/assets/82d091ba-0f6e-4b36-85e4-f4333e5ab727

### CODE REVIEW

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  mongodb:
+    image: "mongo:7-jammy"
+    container_name: mongodb
+    environment:
+      - PUID=1000
+      - PGID=1000
+    volumes:
+      - db:/data/db
+    ports:
+      - 27017:27017
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:12]: is not necessary to expose the ports to the host, remove this

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  minio:
+    image: "minio/minio"
+    command: server /data --address ":9000" --console-address ":9001"
+    ports:
+      - 9000:9000
+      - 9001:9001
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:21]: is not necessary to expose the ports to the host, remove this

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  account:
+    image: hardcoreeng/account:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+    ports:
+      - 3001:3001
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:56]: is not necessary to expose the ports to the host, remove this

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    networks:
+      - dokploy-network
+  elastic:
+    image: "elasticsearch:7.14.2"
+    command: |
+      /bin/sh -c "./bin/elasticsearch-plugin list | grep -q ingest-attachment || yes | ./bin/elasticsearch-plugin install --silent ingest-attachment;
+      /usr/local/bin/docker-entrypoint.sh eswrapper"
+    volumes:
+      - elastic:/usr/share/elasticsearch/data
+    ports:
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:34]: is not necessary to expose the ports to the host, remove this

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+    networks:
+      - dokploy-network
+  collaborator:
+    image: hardcoreeng/collaborator:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+      - transactor
+    ports:
+      - 3078:3078
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:112]: is not necessary to expose the ports to the host, remove this

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  front:
+    image: hardcoreeng/front:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+      - elastic
+      - collaborator
+      - transactor
+    ports:
+      - 8087:8080
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:82]: is not necessary to expose the ports to the host, remove this

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - REKONI_URL=http://rekoni:4004
+      - FRONT_URL=http://${SERVER_ADDRESS}:8087
+      - SERVER_PROVIDER=ws
+      - ACCOUNTS_URL=http://account:3001
+      - LAST_NAME_FIRST=true
+      - UPLOAD_URL=http://${SERVER_ADDRESS}/files
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:155]: is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+      - 4004:4004
+    environment:
+      - SECRET=${SERVER_SECRET}
+    deploy:
+      resources:
+        limits:
+          memory: 500M
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:168]: is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+      - SECRET=${SERVER_SECRET}
+      - ACCOUNTS_URL=http://account:3001
+      - UPLOAD_URL=/files
+      - MONGO_URL=mongodb://mongodb:27017
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:124]: is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - FRONT_URL=http://front:8080
+      - INIT_WORKSPACE=demo-tracker
+      - MODEL_ENABLED=*
+      - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3001
+      - ACCOUNT_PORT=3001
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:72]: is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network

File: apps/dokploy/templates/huly/docker-compose.yml [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - MONGO_URL=mongodb://mongodb:27017
+      - TITLE=Huly Self Hosted
+      - DEFAULT_LANGUAGE=en
+      - LAST_NAME_FIRST=true
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```
Comment (Siumauricio) [inline apps/dokploy/templates/huly/docker-compose.yml:104]: is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network

**Review by Siumauricio — CHANGES_REQUESTED** [review]

**Comment by bhaleraosaurabh** [comment]
Facing issues as there are issues connecting to the application after resolving the PR comments. I'll re-raise the PR once it's fixed

**Pull request closed without merging**